[Bulk] Validation - Missing descriptions (#1474)
dotnet/docs-desktop: dotnet-desktop-guide/framework/wpf/advanced/printing-and-print-system-management.md

---
title: "Printing and Print System Management"

> description: Learn about printing and print system management, which were introduced in Windows Vista and the Microsoft .NET Framework.

Reviewer: ```suggestion
description: Learn about printing and print system management in Windows Presentation Foundation (WPF).
```
